pkg/apis/rook.io/v1alpha2/types.go

```diff
@@ -67,6 +67,8 @@ type Selection struct {
 	Devices []Device `json:"devices,omitempty"`
 
 	Directories []Directory `json:"directories,omitempty"`
+	// PersistentVolumeClaims to use as storage
+	VolumeClaimTemplates []v1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
```

Comment:
did you and @galexrt figure out how to get this volumeClaimTemplates change in smoothly to the common rook.io types, without having to also get all the Ceph specific changes in too?

Reply:
Yeap, for starters I just include a new field in the StorageSpec and all the Ceph logic will come with @galexrt PR.

Reply:
This has been merged to master now with https://github.com/rook/rook/pull/2309, can you make sure that there's no conflict and you are using what's in master now?